### PR TITLE
Update api-go and sdk-go versions 

### LINKED ISSUE #57: Update neofs-api-go to 1.27.0
Update neofs-api-go and sdk-go, check that it works.

### PR DESCRIPTION
**Pull request by masterSplinter01**
Closes #57 

Updated api-go verstion from 1.26.1 to 1.27.0.
Updated sdk-go version.
Replaced usages of deprecated things from api-go.

### CODE REVIEW

**Review by roman-khimov — DISMISSED** [review]

**Comment by roman-khimov** [comment]
Huh, linter is not happy.

**Comment by roman-khimov** [comment]
We need to merge nspcc-dev/neofs-sdk-go#8 first and then use this updated sdk-go.

**Comment by masterSplinter01** [comment]
Fixed

File: go.mod [hunk 1]
```diff
@@ -5,9 +5,9 @@ go 1.16
 require (
 	github.com/fasthttp/router v1.3.5
 	github.com/mr-tron/base58 v1.1.3 // indirect
-	github.com/nspcc-dev/neofs-api-go v1.26.1
+	github.com/nspcc-dev/neofs-api-go v1.27.0
 	github.com/nspcc-dev/neofs-crypto v0.3.0
-	github.com/nspcc-dev/neofs-sdk-go v0.0.0-20210531182150-2f63343bda48
+	github.com/nspcc-dev/neofs-sdk-go v0.0.0-20210603182319-5785062aa943
```
Comment (roman-khimov) [inline go.mod:10]: It should point at f16d38c7b92af22d4698bb459cbd768c4562399c now.

Comment (masterSplinter01) [reply go.mod]: Sorry, forgot to update it. 
Fixed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**